feg/gateway/services/session_proxy/credit_control/gy/gy_client.go

```diff
@@ -300,8 +300,6 @@ func getServiceInfoAvp(server *diameter.DiameterServerConfig, request *CreditCon
 	psInfoAvps := []*diam.AVP{
 		// Set PDP Type as IPV4(0)
 		diam.NewAVP(avp.TGPPPDPType, avp.Vbit, diameter.Vendor3GPP, datatype.Enumerated(0)),
```

Comment:
Don't need it for Gy apparently https://www.developingsolutions.com/DiaDict/Dictionary/3GPP-MS-TimeZone.html